src/main/java/org/jabref/gui/frame/JabRefFrame.java

```diff
@@ -230,11 +258,27 @@ private void initLayout() {
         setTop(head);
 
         splitPane.getItems().addAll(tabbedPane);
+        contentPane.getChildren().addAll(welcomePage, splitPane);
         SplitPane.setResizableWithParent(sidePane, false);
         sidePane.widthProperty().addListener(c -> updateSidePane());
         sidePane.getChildren().addListener((InvalidationListener) c -> updateSidePane());
         updateSidePane();
-        setCenter(splitPane);
+        setCenter(contentPane);
+        updateSidePane();
+        updateContent();
+    }
+
+    private void updateContent() {
+        boolean hasOpenDatabases = !stateManager.getOpenDatabases().isEmpty();
+        welcomePage.setVisible(!hasOpenDatabases);
```

Comment:
If the welcome page is the most 'lowest' node in a StackPane, it should automatically be hidden if a library is shown above. This is the whole purpose of a StackPane to deal with this. So there should be no need for messing with visibility stuff here...